src/vue/backend/src/templates/AgentsSettings.vue

```diff
@@ -77,6 +85,15 @@ export default {
         wrapperEnd: {
             type: String,
             default: ''
+        },
+        formDisabled: {
+            type: Boolean,
+            default: false
+        }
+    },
+    computed: {
+        templateClass () {
```

Comment:
Instead of a computed method, you can  have the classes be an object
```
:className="{
    'form-content': true,
    'form-content--disabled': formDisabled
}"
```

*Repeat for all instances